src/fastga/models/aerodynamics/components/digitization/compute_cn_delta_a_correlation_constatnt.py

```diff
@@ -0,0 +1,152 @@
+            aileron_inner_span_ratio = np.clip(aileron_inner_span_ratio, 0.0, 0.91)
+            _LOGGER.warning(
+                "Aileron inboard location is outside of the range in Roskam's book, value clipped"
+            )
+
+        outputs["aileron_correlation_constant"] = (
+            -0.63632822
+            - 0.57304492 * taper_ratio
+            + 0.21707743 * aspect_ratio
+            + 0.10407542 * aileron_inner_span_ratio
+            + 0.61587892 * taper_ratio**2.0
+            + 0.04971284 * taper_ratio * aspect_ratio
+            - 0.20067475 * taper_ratio * aileron_inner_span_ratio
+            - 0.02997066 * aspect_ratio**2.0
+            - 0.00550507 * aspect_ratio * aileron_inner_span_ratio
+            - 0.00622554 * aileron_inner_span_ratio**2.0
+            - 0.23509612 * taper_ratio**3.0
+            - 0.02728276 * taper_ratio**2.0 * aspect_ratio
+            + 0.12690424 * taper_ratio**2.0 * aileron_inner_span_ratio
+            - 0.00050367 * taper_ratio * aspect_ratio**2.0
+            - 0.01379183 * taper_ratio * aspect_ratio * aileron_inner_span_ratio
+            + 0.04289006 * taper_ratio * aileron_inner_span_ratio**2.0
+            + 0.00140254 * aspect_ratio**3.0
+            + 0.00149011 * aspect_ratio**2.0 * aileron_inner_span_ratio
+            - 0.00553262 * aspect_ratio * aileron_inner_span_ratio**2.0
+            - 0.06766704 * aileron_inner_span_ratio**3.0
+        )
+
+    # pylint: disable=missing-function-docstring, unused-argument
+    # Overriding OpenMDAO compute_partials, not all arguments are used
+    def compute_partials(self, inputs, partials, discrete_inputs=None):
+        taper_ratio = inputs["data:geometry:wing:taper_ratio"]
+        aspect_ratio = inputs["data:geometry:wing:aspect_ratio"]
+        span_ratio = inputs["data:geometry:wing:aileron:span_ratio"]
+
+        taper_ratio_clipped = np.clip(taper_ratio, 0.25, 1.0)
+        aspect_ratio_clipped = np.clip(aspect_ratio, 3.0, 8.0)
+        span_ratio_clipped = np.clip(span_ratio, 0.09, 1.0)
+
+        partials["aileron_correlation_constant", "data:geometry:wing:taper_ratio"] = np.where(
+            taper_ratio == np.clip(taper_ratio, 0.25, 1.0),
+            (
+                -0.00050367 * aspect_ratio_clipped**2.0
+                + 0.01379183 * aspect_ratio_clipped * span_ratio_clipped
+                - 0.05456552 * aspect_ratio_clipped * taper_ratio_clipped
+                + 0.03592101 * aspect_ratio_clipped
+                + 0.04289006 * span_ratio_clipped**2.0
+                - 0.25380848 * span_ratio_clipped * taper_ratio_clipped
+                + 0.11489463 * span_ratio_clipped
+                - 0.70528836 * taper_ratio_clipped**2.0
+                + 1.48556632 * taper_ratio_clipped
+                - 0.73082961
+            ),
+            1e-6,
+        )
+
+        partials["aileron_correlation_constant", "data:geometry:wing:aspect_ratio"] = np.where(
+            aspect_ratio == np.clip(aspect_ratio, 3.0, 8.0),
```

Comment:
```suggestion
            aspect_ratio == aspect_ratio_clipped
```
That way the `np.clip` doesn't have to be ran again